Wrappers/Python/test/utils.py

```diff
@@ -57,12 +57,16 @@ def initialise_tests():
     subprocess.check_output('nvidia-smi')
     has_nvidia = True
 except:
+    if os.environ.get("TESTS_FORCE_GPU", ""):
```

Comment:
@gfardell would you prefer this env var to be documented?

Reply:
It probably should be somewhere :D I don't know where though. I guess a "Running unit tests" section in the contributor guide?

Reply:
added/updated the existing readmes for consistency. Moving to dev guide can be a follow-up PR.